README.md

```diff
@@ -239,24 +239,425 @@ One can also do it manually:
 
       $ dotnet new -u Equinox.Templates
 
+<a name="guidance"></a>
```

Comment:
@nordfjord is patterns a better stable anchor than guidance? i.e. Are these actually patterns? antipatterns? pushy guidance?

Reply:
I don't think these count as patterns. I think we're in pushy guidance territory